sqlalchemy_redshift/commands.py

```diff
@@ -425,9 +425,10 @@ def __init__(self, to, data_location, access_key_id=None,
                     '"ignore_header" parameter should be an integer'
                 )
 
-        if format not in self.formats:
+        if format is not None and str(format).upper() not in self.formats:
             raise ValueError('"format" parameter must be one of %s' %
                              self.formats)
+            format = format.upper()
```

Comment:
still calling .upper() twice.

Reply:
We can remove that line entirely since redshift is case-insensitive but I figured we would keep it to have the output be consistent.

Calling upper() twice isn't really an issue.